packages/functions/src/api/service.ts

```diff
@@ -34,7 +32,17 @@ import { Serializer } from '../serializer';
  */
 interface HttpResponse {
   status: number;
-  json: any;
+  json: HttpResponseJson | null;
+}
+
+export interface HttpResponseJson {
```

Comment:
1. Please add a comment to this interface.
2. I don't know why this file has `Http` vs `JSON` capitalization everywhere, but this interface isn't consistent with it. I hesitate to suggest renaming it to `HttpResponseJSON`, though. That's too horrible. :-(
3. In the comment, I would explicitly say that this interface is to make functions that would otherwise take `{}` able to access the possible elements in the body more easily. This interface doesn't actually make any stronger type guarantee than `{}` would.

I don't really love this interface, but I can't think of any nicer alternative.

Reply:
Updated the interface name to `HttpResponseBody`. WDYT?

Reply:
Very clever. I love it.